solutions/USACO dec20-cowntagion.mdx

```diff
@@ -2,14 +2,69 @@
 id: usaco-1062
 source: USACO Silver 2020 December
 title: Cowntagion
-author: Kevin Sheng
+author: Kevin Sheng (Java, Python), Tanish Tyagi (C++)
 ---
 
 [Official Editorial](http://www.usaco.org/current/data/sol_prob1_silver_dec20.html)
 
```

Comment:
Imo, you should add your written solution for the DFS solution of this problem. Explain how you used DFS to solve this problem.

Reply:
Sounds good

Reply:
Is the official solution particularly bad?

Reply:
If it's the same solution, I'd rather not rewrite it unless we can significantly increase its clarity/quality.

Reply:
I just thought that using dfs was more intuitive and simpler than the official solution.

Reply:
Ok